plugins/modules/zabbix_user.py

```diff
@@ -93,28 +89,27 @@
         description:
             - Whether to enable auto-login.
             - If enable autologin, cannot enable autologout.
-        default: false
         type: bool
     autologout:
         description:
             - User session life time in seconds. If set to 0, the session will never expire.
             - If enable autologout, cannot enable autologin.
-        default: '0'
         type: str
     refresh:
         description:
             - Automatic refresh period in seconds.
+            - The default value will be removed at the version 2.0.0.
```

Comment:
Although I understand why you wanted to do it like this, I think that we can simply omit default value for `refresh` and `rows_per_page` as module defaults it to exactly the same values as Zabbix API itself, thus if we simply pass `None` via API call no backwards compatibility should be broken and it will simplify a few things for us (a.k.a not needing to change this further with 2.0.0)

Reply:
I think that this is the same issue as the above.  
So please see the above my comment.